### PR TITLE
[cloud-provider-yandex] assign route table to existing subnets

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>

## Description

Updated clusterConfiguraiton doc for cloud provider yandex.
## Why do we need it, and what problem does it solve?

Added doc warning about manual binding route table to predefined subnets.
## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cloud-provider-yandex
type: chore
summary: Added doc warning about manual binding route table to predefined subnets.
impact_level: low
```

### CODE REVIEW

File: candi/cloud-providers/yandex/openapi/cluster_configuration.yaml [hunk 1]
```diff
@@ -246,6 +246,7 @@ apiVersions:
           ru-central1-b: e2lu8r1tbbtryhdpa9ro
           ru-central1-c: e2lu8r1tbbtryhdpa9ro
           ```
+          !!!Warning!!! In this case, we must manually bind the route table to the subnet !
```
Comment (konstantin-axenov) [inline candi/cloud-providers/yandex/openapi/cluster_configuration.yaml:249]:
```suggestion
          !!!Warning!!! Deckhouse will create route tabale that must be manually attached to these subnets!
```

File: candi/cloud-providers/yandex/openapi/doc-ru-cluster_configuration.yaml [hunk 2]
```diff
@@ -112,6 +112,7 @@ apiVersions:
           ru-central1-b: e2lu8r1tbbtryhdpa9ro
           ru-central1-c: e2lu8r1tbbtryhdpa9ro
           ```
+          !!!Внимание!!! Необходимо вручную привязать route-table к сабнетам !
```
Comment (konstantin-axenov) [inline candi/cloud-providers/yandex/openapi/doc-ru-cluster_configuration.yaml:115]:
```suggestion
          !!!Внимание!!! Deckhouse создаст таблицу маршрутизации, которую необходимо вручную привязать к указанными подсетям!
```

File: candi/cloud-providers/yandex/openapi/cluster_configuration.yaml [hunk 3]
```diff
@@ -246,6 +246,7 @@ apiVersions:
           ru-central1-b: e2lu8r1tbbtryhdpa9ro
           ru-central1-c: e2lu8r1tbbtryhdpa9ro
           ```
+          !!!Warning!!! Deckhouse will create route tabale that must be manually attached to these subnets!
```
Comment (RomanenkoDenys) [inline candi/cloud-providers/yandex/openapi/cluster_configuration.yaml:249]:
```suggestion
          !!!Warning!!! Deckhouse will create route table that must be manually attached to these subnets!
```

**Review by konstantin-axenov — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @konstantin-axenov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Backport failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/3240775612) for details.

**Pull request merged**